tests/integration/test_api_resources.py

```diff
@@ -301,13 +294,15 @@ def test_local_copy(self):
         response = self.f.copy(effects='resize/50x/')
         self.assertEqual('file', response['type'])
 
-    def test_remote_copy(self):
+    @mock.patch('pyuploadcare.api_resources.rest_request')
+    def test_remote_copy(self, rest_request):
```

Comment:
Btw we already testing it here: https://github.com/uploadcare/pyuploadcare/blob/master/tests/functional/test_api_resources.py#L167-L198

Then i think we can drop the `test_remote_copy(self, rest_request)` now isn't?